src/Entity/Checklist.php

```diff
@@ -23,21 +22,19 @@ class Checklist
 
     /**
      * @ORM\ManyToOne(targetEntity=Project::class, inversedBy="checklists")
-     * @var \App\Entity\Project|null
      */
     private ?Project $project;
 
     /**
      * @ORM\ManyToOne(targetEntity=Checkbox::class)
-     * @var \App\Entity\Checkbox|null
      */
     private ?Checkbox $checkbox;
 
     /**
      * @ORM\Column(type="datetime", nullable=true)
      * @var DateTimeInterface|null
      */
-    private $isComplete;
+    private $completedAt;
```

Comment:
This convention naming for DateTime properties:

- updatedAt
- deletedAt
- pukedAt